master/getting-started/kubernetes/host-endpoints.md

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
This is a doc about host endpoints, but nowhere mentions any host endpoint config.  Would it be worth adding an example HEP yaml doc?

Also I assume this is for non-KDD backend, *and* the yaml docs below are calico yamls not k8s ones?